client/material/tabs.tsx

```diff
@@ -151,13 +151,22 @@ export function Tabs<T>({ children, activeTab, onChange, className }: TabsProps<
         return child
       }
 
-      const hotkeys = [{ keyCode: KEY_NUMBERS[i], ctrlKey: true }]
+      const childHotkeys = (child.props as TabItemProps<T>).hotkeys
+      const hotkeys: HotkeyProp[] = []
+
+      if (Array.isArray(childHotkeys)) {
```

Comment:
This is a weird way to check if something is undefined :)

Reply:
An easier way to do this is just change the above like to `const hotkeys = (childHotkeys ?? []).slice()`